recipe/meta.yaml

```
@@ -18,14 +18,18 @@ build:
   skip: True  # [py<38]
   # Dropping ppc because of various build issues
   skip: True  # [(linux and ppc64le)]
-  # Only building for osx-arm64 GPU in this PR
-  skip: True  # [not (osx and arm64)]
+  # Only building for linux-amd64 GPU in this PR
```

```suggestion
  # Only building for linux-aarch64 GPU in this PR
```
Can be ignored. Just for correction